public/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="images/icon.png" />
+    <link rel="icon" href="assets/icon.png" />
```

Comment:
Address should be relative to where the index.html is located (public folder). It said `images` earlier because that folder is within the public folder.